pkg/controller/jobframework/reconciler.go

```diff
@@ -363,6 +362,15 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, nil
 	}
 
+	if !ptr.Deref(wl.Spec.Active, true) {
+		workload.SetEvictedCondition(wl, kueue.WorkloadEvictedByAcitveField, "spec.active is set to false")
+		err := workload.ApplyAdmissionStatus(ctx, r.client, wl, true)
```

Comment:
Do we still need to apply admission status here or will it be covered in the next sync loop?

Reply:
We do. This will update the condition on the apiserver